instructure/canvas-ios
# Convert old request to get students on dashboard to a useCase

refs: MBL-13921
affects: Parent
release note: none

Test plan
---------------
make sure students still show up on dashboard and you can select different students

1. login with an observer with 1 student only (observer2 / twilson ) - make sure menu does not show
2. login with an observer with 0 students (grichards / o2 )  - make sure the not a teacher modal is shown
3. login with an observer with > 1 student (observer1 / twilson ) - menu should show
